src/Infrastructure/Translation/TranslatorInterface.php

```diff
@@ -8,7 +8,7 @@ interface TranslatorInterface {
 
 	/**
 	 * @param string $messageKey
-	 * @param array<string, scalar> $parameters
+	 * @param array<string, string|float|null> $parameters
```

Comment:
```suggestion
	 * @param array<string, scalar|null> $parameters
```

If it was scalar before, I'd rather not risk an error when passing an integer or boolean.

Reply:
@gbirke PHPStan was giving me an error on this line, to fix it I made the changes.

If I change it back to `scalar` the error will come back, what to do in that case??

Reply:
I have made the changes suggested by you.